### PR TITLE
Docs: How to include manual tests into the next release

### PR DESCRIPTION
**Pull request by b1zzu**
# Description

Document how to include manual test cases into the next release

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: test-cases/README.md [hunk 1]
```diff
@@ -37,6 +52,43 @@ npm run prettier
 
 Commit everything and open a new PR.
 
+## How to include a manual test case in the next release
+
+> Attention: all changes need to be manually pushed to master or to the release branch
+
+By default we don't execute all manual tests on each release, therefore if we know we have made a change to a component and a specific test should be executed verify the change or prevent regressions we must manually add the test case to the next release.
```
Comment (trepel) [inline test-cases/README.md:59]: replace `executed verify` with `executed to verify`

File: test-cases/README.md [hunk 2]
```diff
@@ -37,6 +52,43 @@ npm run prettier
...
+
+To do it we need to add the to the `targets` the next version, for example if the last released version was `2.7.0` then the next release version would be `2.8.0`.
```
Comment (trepel) [inline test-cases/README.md:61]: `the to the` -> I would change the whole sentence:
"To do that the next version needs to added to the `targets`,..."

File: test-cases/README.md (hunk 1 above)
Comment (trepel) [inline test-cases/README.md:59]:
```suggestion
By default we don't execute all manual tests on each release, therefore if we know we have made a change to a component and a specific test should be executed to verify the change or prevent regressions we must manually add the test case to the next release.
```

File: test-cases/README.md (hunk 2 above)
Comment (trepel) [inline test-cases/README.md:61]:
```suggestion
To do that the next version needs to be added to the `targets`, for example if the last released version was `2.7.0` then the next release version would be `2.8.0`.
```

File: test-cases/README.md [hunk 3]
```diff
@@ -37,6 +52,43 @@ npm run prettier
 
 Commit everything and open a new PR.
 
+## How to include a manual test case in the next release
+
+> Attention: all changes need to be manually pushed to master or to the release branch
+
+By default we don't execute all manual tests on each release, therefore if we know we have made a change to a component and a specific test should be executed to verify the change or prevent regressions we must manually add the test case to the next release.
+
+To do that the next version needs to added to the `targets`, for example if the last released version was `2.7.0` then the next release version would be `2.8.0`.
```
Comment (trepel) [inline test-cases/README.md:61]:
```suggestion
To do that the next version needs to be added to the `targets`, for example if the last released version was `2.7.0` then the next release version would be `2.8.0`.
```

**Review by trepel — CHANGES_REQUESTED** [review]

File: test-cases/README.md (hunk 3 above)
Comment (trepel) [reply test-cases/README.md:61]: Sorry @b1zzu , in my suggestion there was a mistake :+1: Once this is fixed, I am happy to lgtm and approve this one

Comment (b1zzu) [reply test-cases/README.md]: np, there was no rush to merge it

**Comment by b1zzu** [comment]
@tremes should be ready now

**Pull request merged**